MashUp-iOS/MashUp-iOS/Source/Screen/Seminar Schedule/View/AttendenceTimeAndCheckView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  AttendenceTimeAndCheckView.swift
+//  MashUp-iOS
+//
+//  Created by 이문정 on 2022/06/18.
+//  Copyright © 2022 Mash Up Corp. All rights reserved.
+//
+
+import SnapKit
+import Then
+import UIKit
+import MashUp_Core
+
+class AttendenceTimeAndCheckView: UIView {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+        setupUI()
+    }
+    private let checkImageView = UIImageView()
+    private let seminarSectionLabel = UILabel()
+    private let attendenceLabel = UILabel()
+    private let timeLabel = UILabel()
+}
+
+extension AttendenceTimeAndCheckView {
+    private func setupUI() {
+        self.setupLayout()
+        self.setupAttribute()
+    }
+    
+    private func setupAttribute() {
+        layoutIfNeeded()
+        self.checkImageView.do {
+            $0.backgroundColor = .green500
+            $0.layer.cornerRadius = $0.bounds.width / 2
+            $0.clipsToBounds = true
+        }
+        self.seminarSectionLabel.do {
+            $0.text = "1부"
+            $0.textColor = .gray600
+            $0.font = .pretendardFont(weight: .regular, size: 12)
+        }
+        self.attendenceLabel.do {
+            $0.text = "출석"
+            $0.textColor = .gray600
+            $0.font = .pretendardFont(weight: .semiBold, size: 16)
+        }
+        self.timeLabel.do {
+            $0.text = "오후 3:25"
+            $0.textColor = .gray500
+            $0.font = .pretendardFont(weight: .regular, size: 14)
+        }
+    }
+    
+    private func setupLayout() {
+        self.do {
+            $0.addSubview(self.checkImageView)
+            $0.addSubview(self.seminarSectionLabel)
+            $0.addSubview(self.attendenceLabel)
+            $0.addSubview(self.timeLabel)
+        }
+        
+        self.checkImageView.snp.makeConstraints {
+            $0.top.equalToSuperview().inset(0)
+            $0.leading.equalToSuperview().inset(0)
+            $0.bottom.equalToSuperview().inset(0)
```

Comment:
inset(0) 은 명시적으로 표시하지 않아도 될 것 같습니답 🙇‍♂️
```suggestion
            $0.top.leading.bottom.equalToSuperview()
```